app/src/models/merge.ts

```diff
@@ -36,3 +36,12 @@ export interface IMergeUnsupported {
 }
 
 export type MergeResult = IMergeSuccess | IMergeError | IMergeUnsupported
+
+export enum MergeSource {
+  BranchDropDown,
```

Comment:
:art: convert this to a string enum to simplify the generated JS [[link](https://www.typescriptlang.org/play/#src=export%20enum%20MergeSourceNumber%20%7B%0D%0A%20%20BranchDropDown%2C%0D%0A%20%20MergeIntoCurrentBranchMenuItem%2C%0D%0A%20%20UpdateBranchMenuItem%2C%0D%0A%20%20Compare%2C%0D%0A%20%20DivergingBranchNotification%2C%0D%0A%20%20NewCommitsBanner%2C%0D%0A%7D%0D%0A%0D%0Aexport%20enum%20MergeSourceString%20%7B%0D%0A%20%20BranchDropDown%20%3D%20'BranchDropDown'%2C%0D%0A%20%20MergeIntoCurrentBranchMenuItem%20%3D%20'UpdateBranchMenuItem'%2C%0D%0A%20%20UpdateBranchMenuItem%20%3D%20'UpdateBranchMenuItem'%2C%0D%0A%20%20Compare%20%3D%20'Compare'%2C%0D%0A%20%20DivergingBranchNotification%20%3D%20'DivergingBranchNotification'%2C%0D%0A%20%20NewCommitsBanner%20%3D%20'NewCommitsBanner'%2C%0D%0A%7D%0D%0A)]